running-recipes/running-rewrite-on-a-gradle-project-without-modifying-the-build.md

```diff
@@ -39,10 +44,15 @@ rootProject {
     }
 }
 ```
+{% endcode %}
+
+In the `rootProject` block, we specify a dependency that contains OpenRewrite spring recipes (`rewrite-spring`). If you wanted, you could also define a `rewrite` section inside of the `rootProject` that has elements like `activeRecipe` or `activeStyle`. In general, though, it's better to [use the command line to specify the recipes or styles](#step-4-run-the-recipe) so that you can keep your init script fairly generic. 
```

Comment:
```suggestion
In the `rootProject` block, we specify a dependency that contains OpenRewrite Spring recipes (`rewrite-spring`). If you wanted, you could also define a `rewrite` section inside of the `rootProject` that has elements like `activeRecipe` or `activeStyle`. In general, though, it's better to [use the command line to specify the recipes or styles](#step-4-run-the-recipe) so that you can keep your init script fairly generic. 
```